### ISSUE TITLE
converting v1 to track foods should run typeof check to allow falsey values of same type

### ISSUE THREAD

**Issue by owen-d** [nutritionix/nutritionix-api-data-utilities#6]
i.e. converting a food with 0 calories should not default to null calories.

change needed [here](https://github.com/nutritionix/nutritionix-api-data-utilities/blob/master/src/index.js#L84)

**Comment by Yurko-Fedoriv** [comment]
@vgupta16 Can you please check on this? I guess we need to replace
```js
return baseTrackObj.hasOwnProperty(key) && val;
```

with 

```js
return baseTrackObj.hasOwnProperty(key) && (val || val === 0);
```

What do you think?

**Comment by vgupta16** [comment]
@Yurko-Fedoriv That looks good to me

**Comment by Yurko-Fedoriv** [comment]
@vgupta16 released [2.2.1](https://github.com/nutritionix/nutritionix-api-data-utilities/tree/2.2.1)
please let me know if there are any issues.

**Comment by vgupta16** [comment]
Thanks @Yurko-Fedoriv
